src/Mod/Start/Gui/DlgStartPreferences.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>482</width>
-    <height>795</height>
+    <width>537</width>
+    <height>942</height>
```

Comment:
150 px more height for an option seems wrong.

However in the application the height is recalculated anyway. So I will merge and have a look at the ui file later.

Reply:
yes at some point that page will need to be splitted I guess. For the time being, I think it still think it looks best as a single, long list I guess